twisted_tools/scripts/forcebuild.py

```diff
@@ -1,9 +1,9 @@
 """
-Force the Twisted buildmaster to run a builds on all supported builders for
-a particular branch.
+Command line adapter for L{buildot.forceBuild}.
```

Comment:
My intention was to remove the duplicate docstring. I tried a small  change now.
Please suggest a better docstring.